showcase/app/templates/page-components/form/radio.hbs

```diff
@@ -390,6 +390,7 @@
       <Hds::Form::Radio::Group @name="control-booleans" as |G|>
         {{#let (array true false) as |bools|}}
           {{#each bools as |bool|}}
+            {{! @glint-expect-error }}
```

Comment:
@hashicorp/hds-engineering I remember we added this example because a consumer found a bug with Radios if they set the value to a boolean. The current component only allows a string for the id and value. Should we change the component type to allow strings or booleans?

Reply:
Yeah, I remember the same. Personally, I would not change the component, but I would add a comment here in the `@glint-expect-error` declaration, explaining why we're testing also this use case